docs_src/microservices/support/cronScheduler/GettingStarted.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Support Cron Scheduler - Getting Started
+---
+
+# Support Cron Scheduler - Getting Started
+
+Support Cron Scheduler is one of the EdgeX Support Services which aims to replace the old Support Scheduler serivce.
```

Comment:
add a warning to the old Scheduler service page to state this service is deprecated
https://github.com/edgexfoundry/edgex-docs/blob/5d1b31cf3bb2c2aa006b6d8f178217367287c2a6/docs_src/microservices/system-management/Ch_SystemManagement.md?plain=1#L5-L6